documentation/asciidoc/accessories/camera/libcamera_apps_getting_started.adoc

```diff
@@ -0,0 +1,60 @@
+=== Getting Started
+
+==== Using the camera for the first time
+
+When running a Raspberry Pi OS based on _Bullseye_ or later, the 5 basic `libcamera-apps` are already installed. In this case, official Raspberry Pi cameras will also be detected and enabled automatically.
+
+You can check that everything is working by entering:
+
+[,bash]
+----
+libcamera-hello
+----
+
+You should see a camera preview window for about 5 seconds.
+
+Normally there is nothing further to do, but if you do need to alter the configuration of your Raspberry Pi, please consult the next section.
+
+NOTE: Pi 3 and older devices may not by default be using the correct display driver. Refer to the `/boot/config.txt` file and ensure that either `dtoverlay=vc4-fkms-v3d` or `dtoverlay=vc4-kms-v3d` is currently active. Please reboot if you needed to change this.
+
+==== If you do need to alter the configuration
+
+You may need to alter the camera configuration in your `/boot/config.txt` file if:
+
+* You are using a 3rd party camera (the manufacturer's instructions should explain the changes you need to make).
+
+* You are using an official Raspberry Pi camera but wish to use a non-standard driver/overlay.
+
+* You are using an older version (_Buster_ or earlier) of the operating system (in which case we would recommend upgrading first if this is possible).
```

Comment:
Maybe we need instructions somewhere that a user can use to determine if they're running a 'Buster' or a 'Bullseye' image?

Reply:
Yup and that should go here, https://www.raspberrypi.org/documentation/computers/os.html#updating-and-upgrading-raspberry-pi-os, I guess right at the start of the updating and upgrading bit.

Reply:
Does `cat /etc/os-release` still work?